rclcpp_action/include/rclcpp_action/client.hpp

```diff
@@ -317,7 +317,7 @@ class Client : public ClientBase
     rclcpp::node_interfaces::NodeGraphInterface::SharedPtr node_graph,
     rclcpp::node_interfaces::NodeLoggingInterface::SharedPtr node_logging,
     const std::string & action_name,
-    const rcl_action_client_options_t client_options = rcl_action_client_get_default_options()
+    const rcl_action_client_options_t & client_options
```

Comment:
@fujitatomoya there's no need to remove the default here.

Reply:
change it back then. i knew that, i was just trying to keep it consistent with server implementation.